WS2012R2/lisa/setupscripts/MultipleReboot.ps1

```diff
@@ -298,6 +330,14 @@ While( -not (TestPort $vmIPAddr) )
     Start-Sleep 5
 }
 
+
+if ((Get-VMEvent -VMName $vmName -HvServer $hvServer -StartTime $testStartTime `
+        -EventCode 18602 -RetryCount 2 -RetryInterval 1)) {
+    Write-Output "Error: VM $vmName triggered a critical event 18602 on the host."| `
+        Out-File -Append $summaryLog
+    return $False
+}
+
 $retVal = $true
 Write-Output "VM rebooted $bootcount times successfully" | Out-File -Append $summaryLog
```

Comment:
might be worth also adding
Write-Output "Event ID 18602 for VM boot failure was not detected" | Out-File -Append $summaryLog